### PR TITLE
Improve de locale

### PR DESCRIPTION
**Pull request by 1st8**
Probably more to come when I start using MUC and other stuff, but this fixes a lot of mistakes in german and adds some translations.

Cheers

### CODE REVIEW

File: locale/de/LC_MESSAGES/converse.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
          ],
          "Click to hide these contacts": [
             null,
-            "Hier klicken um diesen Kontakte zu verstecken"
+            "Hier klicken um diesen Kontakt zu verstecken"
```
Comment (jcbrand) [inline locale/de/LC_MESSAGES/converse.json]: I think this is a mistake, it should still be plural.

**Comment by jcbrand** [comment]
Thanks!

**Comment by 1st8** [comment]
You're welcome! 

The "Click to hide these contacts" translation should be plural indeed. 
Since you merged it already, do you want me to prepare a second PR?

**Comment by jcbrand** [comment]
@1st8 No don't worry, I'll quickly change it myself.

**Pull request merged**